pkg/virt-controller/watch/export/export.go

```diff
@@ -978,6 +988,40 @@ func (ctrl *VMExportController) updateVMExportPvcStatus(vmExport *exportv1.Virtu
 	return requeue, nil
 }
 
+func (ctrl *VMExportController) updateVMSnapshotExportStatus(vmExportCopy *exportv1.VirtualMachineExport, pvcs []*corev1.PersistentVolumeClaim) error {
```

Comment:
I think it's a little awkward that `handleIsVmSnapshot` calls `updateVMExportPvcStatus` which then checks if it is a snapshot export and calls `updateVMSnapshotExportStatus`.  Why not have `handleIsVmSnapshot` call `updateVMSnapshotExportStatus` (and you can pass the snapshot as a param) which then calls `updateVMExportPvcStatus` or some other new shared function

Basically, the controller should only have to switch on export source type once